geometry_msgs/PoseWithCovariance pose_w_cov
float64 ellipse_area
float64 epsilon
